src/main/java/apoc/export/util/ExportFormat.java

```diff
@@ -36,7 +37,7 @@ public static final ExportFormat fromString(String format) {
                 }
             }
         }
-        return NEO4J_SHELL;
+        return CYPHER;
```

Comment:
If we're changing the default in a .dot release, we have to clearly communicate it, or leave it for 3.2